build.gradle.kts

```diff
@@ -12,7 +12,7 @@ buildscript {
 
 plugins {
     kotlin("jvm")
-    id("org.jetbrains.intellij") version "1.12.0"
+    id("org.jetbrains.intellij") version "1.13.2"
```

Comment:
Why pick this specific version instead of 1.14.0 or 1.15.0?

Reply:
For now we can use the version 1.13.2 to unblock the current release. There will be more configuration issues to solve than expected when changing the version to 1.14.0 or 1.15.0. I added a `TODO` in the next commit.